packages/svelte/tests/runtime-legacy/samples/escaped-attr-2/_config.js

```diff
@@ -0,0 +1,7 @@
+import { test } from '../../test';
+
+export default test({
+	test({ assert, logs }) {
+		assert.deepEqual(logs, []);
```

Comment:
Is this a good test? Have you checked that JSDOM will run the `console.log()` without this fix?

Reply:
Yep those tests fail :)